go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aquasecurity/trivy
 go 1.13
 
 require (
-	github.com/aquasecurity/fanal v0.0.0-20200221125056-947c2a5bb130
+	github.com/aquasecurity/fanal v0.0.0-20200227041215-32a05c8b0427
```

Comment:
Note to myself: Before we merge this in, let's merge the fanal PR first and then update our dependency to the head of fanal rather than a feature branch like here.